README.md

```diff
@@ -1,175 +1,529 @@
+```bash
+uv pip install postgres-mcp
+```
+
+If you need to install `uv`, see the [uv installation instructions](https://docs.astral.sh/uv/getting-started/installation/).
+
+#### Option 2: Using Docker
 
-Example with automatic localhost remapping:
+Pull the Postgres Pro MCP server Docker image.
+This image contains all necessary dependencies, providing a reliable way to run Postgres Pro in a variety of environments.
 
 ```bash
-docker run -it --rm crystaldba/postgres-mcp "postgres://user:password@localhost:5432/dbname"
+docker pull crystaldba/postgres-mcp
 ```
 
-### Setting Up with AI Assistants
+### Configure Your AI Assistant
 
-#### Claude Desktop
+We provide full instructions for configuring Postgres Pro with Claude Desktop.
+Many MCP clients have similar configuration files, you can adapt these steps to work with the client of your choice.
 
-Configure Claude Desktop to use postgres-mcp:
+#### Claude Desktop Configuration
 
-1. Edit your Claude Desktop configuration file:
+You will need to edit the Claude Desktop configuration file to add Postgres Pro.
+The location of this file depends on your operating system:
+- MacOS: `~/Library/Application Support/Claude/claude_desktop_config.json`
+- Windows: `%APPDATA%/Claude/claude_desktop_config.json`
 
-   - MacOS: `~/Library/Application Support/Claude/claude_desktop_config.json`
-   - Windows: `%APPDATA%/Claude/claude_desktop_config.json`
+You can also use `Settings` menu item in Claude Desktop to locate the configuration file.
 
-2. Add the following to the "mcpServers" section:
+You will now edit the `mcpServers` section of the configuration file.
 
-If you want "restricted mode" -- all queries are wrapped in a read-only transaction, with a query timeout of 30 seconds, and additional scrubbing:
+##### If you are using `pipx`
 
 ```json
 {
   "mcpServers": {
     "postgres": {
-      "command": "docker",
+      "command": "postgres-mcp",
       "args": [
-        "run",
-        "-i",
-        "--rm",
-        "crystaldba/postgres-mcp",
         "postgresql://username:password@localhost:5432/dbname",
-        "--access-mode=restricted"
+        "--access-mode=unrestricted"
+      ]
     }
   }
 }
 ```
 
-If you want "unrestricted mode" -- you can do anything:
+##### If you are using `uv`
 
 ```json
 {
   "mcpServers": {
     "postgres": {
-      "command": "docker",
+      "command": "uv",
       "args": [
         "run",
-        "-i",
-        "--rm",
-        "crystaldba/postgres-mcp",
-        "postgresql://username:password@localhost:5432/dbname"
+        "postgres-mcp",
+        "postgresql://username:password@localhost:5432/dbname",
+        "--access-mode=unrestricted"
+      ]
     }
   }
 }
 ```
 
-#### Cursor
-
-Configure Cursor to use postgres-mcp:
-
-1. Create or edit `~/.cursor/mcp.json` (or run Ctrl-Shift-P and type "Developer: Open Logs Folder")
-2. Add the same `mcpServers` configuration as above.
+##### If you are using Docker
 
 ```json
 {
```

Comment:
The Docker instructions don't specify how to handle volume mounting if users need persistence

Reply:
I don't think there is any persistence.